### DISCUSSION TITLE
ShowGrouping section has no customizable text.

### DISCUSSION THREAD

**Discussion by michele-bettin** [Megabit/Blazorise#5167] [General]
Before opening it as an issue I would like you to confirm that the ShowGrouping message dialog text "Drag a column header here to group by that column" is not customizable because there is not customizable EditTemplate...
I try with HeaderGroupCaptionTemplate but doesn't work

**Comment by David-Moreira** [comment]
If I remember correctly you need to provide your own Localizers with what you would like to be overriden with. There should be a localizer there for every text that can be changed.
![image](https://github.com/Megabit/Blazorise/assets/22283161/8de6da97-fb92-4bbe-a82c-77e135aaba59)
